include/alignment.h

```diff
@@ -8,9 +8,15 @@
 #define ALIGN256(val) (((val) + 0xFF) & ~0xFF)
 
 #ifdef __GNUC__
-#define ALIGNED8 __attribute__ ((aligned (8)))
+#define ALIGNED4  __attribute__ ((aligned (4)))
+#define ALIGNED8  __attribute__ ((aligned (8)))
+#define ALIGNED16 __attribute__ ((aligned (16)))
+#define ALIGNED64 __attribute__ ((aligned (64)))
 #else
+#define ALIGNED4
 #define ALIGNED8
+#define ALIGNED16
+#define ALIGNED64
 #endif
```

Comment:
Why not have a single `ALIGNED(x)` instead?

Reply:
For some reason I thought ALIGNED8 was used sometimes in assets but that just wasn't true at all

Reply:
iirc that was the original intention, but it was never implemented